src/adapter/src/session/vars.rs

```diff
@@ -373,6 +412,15 @@ static EMIT_TRACE_ID_NOTICE: ServerVar<bool> = ServerVar {
     description:
         "Boolean flag indicating whether to send a NOTICE specifying the trace id when available (Materialize).",
     internal: false,
+    safe: true,
+};
+
+static MOCK_AUDIT_EVENT_TIMESTAMP: ServerVar<Option<u64>> = ServerVar {
```

Comment:
I'm not a user of this so don't care either way, but wondering if a timestamp type instead of u64 is more easily usable.

Reply:
Good call, I'll switch it to the `mz_repr::Timestamp` type

Reply:
If that easily parses both ints and timestamps from a `SET` command that'd be great. I don't know if it does offhand?

Reply:
Oooh I see what you mean. I just checked and it does not. I'm not sure if there's a type that can parse both as an epoch timestamp and as a human readable timestamp, unless we roll something ourselves.

@necaris do you have a preference between being able to set the value using an epoch or human readable timestamp? Or would you prefer to be able to do both, in which case I'll try and implement something myself.

Reply:
We're expecting to mostly use this from a test harness so an epoch timestamp is mildly preferable if it's a choice between the two, although ISO-formatting isn't hard either way.

Reply:
I'll keep it as an epoch parsing then. Though I will switch the type to `mz_repr::Timestamp` so it better matches the actual type used in the table.